Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+	<PropertyGroup>
+		<!-- Compile Include="x.cs" doesn't use LangVersion defined in the PropertyGroup -->
+		<LangVersion>9.0</LangVersion>
```

Comment:
I'd prefer to keep the setting either in csproj or in \Build\Props\CodeJam.Default.props

Reply:
The problem is that it doesn't help.
Files such as `<Compile Include="JetBrains.Annotations.cs"/>` don't receive LangVersion 9.0.
I submitted a bug.

It it possible to include a file from Build\Props\... , but then there is no difference between setting it here or including.

Reply:
I simply didn't find other working solution.

Reply:
Aha! Let's remove <LangVersion> from csproj files then

Reply:
Ok.
Btw, you can vote for the issue: https://developercommunity2.visualstudio.com/t/itemgroup-compile-include-doesnt-use-the-same-lang/1255582?from=email&preview=true